utils/frame/benchmarking-cli/src/command.rs

```diff
@@ -110,7 +125,14 @@ impl BenchmarkCmd {
 		let genesis_storage = spec.build_storage()?;
 		let mut changes = Default::default();
 		let cache_size = Some(self.database_cache_size as usize);
-		let state = BenchmarkingState::<BB>::new(genesis_storage, cache_size, self.record_proof)?;
+		let state_with_tracking = BenchmarkingState::<BB>::new(
+			genesis_storage.clone(),
+			cache_size,
+			self.record_proof,
+			true,
```

Comment:
It's not quite obvious what this boolean represents when looking at the code, perhaps create a new enum that is more descriptive?

```rust
enum Tracking { Enable, Disable }
```